### ISSUE TITLE
[Feature Request] Awesome Feature

### ISSUE THREAD

**Issue by VasilisThePikachu** [AsteroidOS/asteroid#232]
**Is your feature request related to a problem? Please describe.**
I recently installed Asteroid on my Huawei watch 2, and so far it's pretty good, Though one feature I'm missing from WearOS is the media controls for music playing on my phone

**Describe the feature you'd like**
The way WearOS achieved this is by showing the media player as you start playing some media on the connected phone, with back/forward track buttons, volume, pause/resume, and the time at the top. And the current Tittle/Artist and cover in the background

**Additional context**
Here is a screenshot from WearOS about how it would look like, I don't know if it's possible but maybe a little heart/like button to like songs/videos may be possible? not sure
![image](https://user-images.githubusercontent.com/34938708/213245223-ca615b81-a733-4487-98d0-a250fa783ccb.png)

**Comment by eLtMosen** [comment]
Sorry for the silence on this. But i picked up your request just now and added a musicButton to the newly designed QuickSetting panel you can pull down from above the homescreen.
That way you would not need to dig out the music app form any launcher and can place the music app link in a place that is more easily accessible.
I will close this issue once the change is merged. If this implementation is not what you intended, please make a new issue with suggestions for changes or additions.
https://github.com/AsteroidOS/asteroid-launcher/commit/2c35df7189217cb1225544a2862585e215ff93cf
